### PR TITLE
DT-1590 Add HMPPS Domain Events Topic

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-registers.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+resource "kubernetes_secret" "hmpps-registers" {
+  metadata {
+    name      = "hmpps-domain-events-topic"
+    namespace = "hmpps-registers-dev"
```
Comment (Mjwillis) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-registers.tf:4]: pushing creds to other namespace

Comment (poornima-krishnasamy) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-registers.tf:4]: @Mjwillis , I dont think you can push creds secrets to another namespace as they share different terraform states.

Comment (Mjwillis) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-registers.tf:4]: yes you can @poornima-krishnasamy

Comment (Mjwillis) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-registers.tf:4]: We already do this in offender-events-dev namespace

Comment (poornima-krishnasamy) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-registers.tf:4]: Thats interesting @Mjwillis. Thanks for the reference.

**Review by poornima-krishnasamy — APPROVED** [review]

**Pull request merged**